File: answers/VrishtiSharma/looping/Q4.py
a = int(input("Enter first no. : "))
b = int(input("Enter second no. : "))
for num in range(a,b + 1):
   if num > 1:
       for i in range(2,num):
           if (num % i) == 0:
               break
       else:
           print(num, end = " ")

File: answers/jahnvit25/Looping/Q4.cpp
#include <stdio.h>
int main() 
{
   int first, second, i, flag;
   printf("Enter the two numbers:\n");
   scanf("%d %d", &first, &second);
   printf("Prime numbers between the two numbers are: ",first,second);
   while (first < second)
   {
       flag = 0;

      if (first <= 1)
      {
         ++first;
         continue;
      }

      for (i = 2; i <= first / 2; ++i) 
      {
          if (first % i == 0)
          {
            flag = 1;
            break;
          }
      }

      if (flag == 0)
      printf("%d ", first);

      ++first;
   }

   return(0);
}
